src/wallet/subcommands/wallet_cmd.rs

```diff
@@ -253,11 +450,106 @@ impl WalletCommands {
                 };
                 let msg = hex::decode(message).context("Message has to be a hex string")?;
 
-                let response = api.wallet_verify(address, msg, signature).await?;
+                let is_valid = backend.wallet_verify(address, msg, signature).await?;
+
+                println!("{is_valid}");
+                Ok(())
+            }
+            Self::Send {
+                from,
+                target_address,
+                amount,
+                gas_feecap,
+                gas_limit,
+                gas_premium,
+            } => {
+                let from: Address = if let Some(from) = from {
+                    StrictAddress::from_str(&from)?.into()
+                } else {
+                    StrictAddress::from_str(&backend.wallet_default_address().await?.context(
+                        "No default wallet address selected. Please set a default address.",
+                    )?)?
+                    .into()
+                };
+
+                let message = Message {
+                    from,
+                    to: StrictAddress::from_str(&target_address)?.into(),
+                    value: amount,
+                    method_num: METHOD_SEND,
+                    gas_limit: gas_limit as u64,
+                    gas_fee_cap: gas_feecap,
+                    gas_premium,
+                    // JANK(aatifsyed): Why are we using a testing build of fvm_shared?
```

Comment:
What do we do with this?

Reply:
It's a wrinkle in `fvm_shared`. I guess we could send a patch upstream.